autogen/agentchat/groupchat.py

```diff
@@ -259,6 +259,20 @@ def _mentioned_agents(self, message_content: Union[str, List], agents: List[Agen
         Returns:
             Dict: a counter for mentioned agents.
         """
+        # This handles the case when the selector gives a wrong response of making function calls
+        # take out content, function_calls, tool_calls to put in one string
+        if isinstance(message_content, dict):
+            tmp_content = ""
+            if message_content.get("content"):
+                tmp_content = message_content["content"]
+            if message_content.get("function_call"):
+                tmp_content += message_content["function_call"]["name"] + message_content["function_call"]["arguments"]
+            if message_content.get("tool_calls"):
+                for tool_call in message_content["tool_calls"]:
+                    if tool_call.get("type") == "function":
+                        tmp_content += tool_call["function"]["name"] + tool_call["function_call"]["arguments"]
```

Comment:
Is this unnecessary if functions and tools are disabled?

Reply:
Yes, I am removing that